modules/serverless-logic-global-config-customization.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-global-config-settings
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-global-config-customization_{context}"]
+= Customization of global configurations
+
+After installing the {ServerlessLogicOperatorName}, you can access the `sonataflow-operator-controllers-config` ConfigMap file in the `sonataflow-operator-system` namespace. This configuration file defines how the operator behaves when it creates new resources in the cluster. However, changes to this configuration will not affect resources that already exist. 
+
+You can modify any of the options within the `controllers_cfg.yaml` key in the `ConfigMap`. 
+
+The following table outlines all the available global configuration options:
+
+[cols=3*,options="header"]
+|===
+|Configuration key
+|Default value
+|Description
+
+|`defaultPvcKanikoSize`
+|`1Gi`
+|The default size of Kaniko PVC when using the internal {ServerlessLogicOperatorName} builder manager.
+
+|`healthFailureThresholdDevMode`
+|`50`
+|How much time (in seconds) to wait for a devmode workflow to start. This information is used for the controller manager to create new devmode containers and setup the healthcheck probes.
+
+|`kanikoDefaultWarmerImageTag`
+|`gcr.io/kaniko-project/warmer:v1.9.0`
+|Default image used internally by the Operator Managed Kaniko builder to create the warmup pods.
+
+|`kanikoExecutorImageTag`
+|`gcr.io/kaniko-project/executor:v1.9.0`
+|Default image used internally by the Operator Managed Kaniko builder to create the executor pods.
+
+|`jobsServicePostgreSQLImageTag`
+|empty
```

Comment:
Value can be found in the handover email.

Reply:
The table will probably look horrible with such long image names in the rows, but I think we should include the values.
Because, in the end, this is what the operator will use by default.

Reply:
Can you please tell me the values here? As I did not receive any handover email.

Reply:
Hi @kaldesai these are the default values:

jobsServicePostgreSQLImageTag: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.35.0

jobsServiceEphemeralImageTag: registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8:1.35.0

dataIndexPostgreSQLImageTag: registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel8:1.35.0

dataIndexEphemeralImageTag: registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel8:1.35.0

sonataFlowBaseBuilderImageTag: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.35.0

sonataFlowDevModeImageTag: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:1.35.0

My suggestion here is to  use a variable with the current version number `versionNumber` (I think it should be defined somewhere already, if not, we can add it), and then, add the values like this:

registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:**{versionNumber}**

Because the image numbers change on every new release. So, in the next release, the images will be 1.36.0 instead of 1.35.0. But all the rest of the names we keep the same.

Reply:
Thank you for providing the default values. I was able to find the **{ServerlessProductVersion}** attribute so I replaced it and now I have added the default values.